bndtools.builder/src/org/bndtools/builder/MarkerSupport.java

```diff
@@ -98,7 +98,7 @@ private void deleteCnfMarkersForThisProject(String markerType) throws CoreExcept
 		IProject cnf = ResourcesPlugin.getWorkspace()
 			.getRoot()
 			.getProject("cnf");
-		if (cnf != null) {
+		if (cnf != null && cnf.exists()) {
```

Comment:
Should this not be `cnf.isOpen()`?

Reply:
I don't know enough about Eclipse's marker handling to know the answer to this question. Checking for existence was sufficient to stop the NPE from happening in the enclosing code if the cnf project was missing. But checking for openness is probably not a bad idea.